### ISSUE TITLE
optimize with fragment.zbuf, not converge 

### ISSUE THREAD

**Issue by JobAtom** [facebookresearch/pytorch3d#1167]
I am generating the zbuf from fragments.zbuf as depth map to optimize my network. However, the loss functions never converge when I put the terms contain zbuf. When I removed the referenced terms, the loss will converge. 

I am thinking the zbuf from fragments.zbuf not actually working. 

I search around the issue and found that some one said changing the padding value( -1 in zbuf) to any positive numbers or 0 to make the loss converge. But those solutions never worked to me. Is there any comments about the zbuf ?

I am using the following raster setting: 
raster_settings = RasterizationSettings(
            image_size=256,
            blur_radius=0.0,
            faces_per_pixel=1
        )

**Comment by gkioxari** [comment]
I recommend moving this question to the GH discussion page to get help from other users.

**Comment by bottler** [comment]
(Note that the problem here is the design of a model so that it trains well, which may be an open research question, rather than the implementation of the design.)
